test/e2e/storage/persistent_volumes-local.go

```diff
@@ -143,7 +159,10 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 		})
 
 		AfterEach(func() {
-			cleanupLocalVolume(config, testVol)
+			if testVol != nil {
+				cleanupLocalVolume := cleanupLocalVolumeMap[testVol.localVolumeType]
+				cleanupLocalVolume(config, testVol)
```

Comment:
Can you deref the function inside a wrapper cleanup method?  Then you can check for nil in there, and also put the PVC cleanup in there too.